curation-migration-datasync/database/dataService.ts

```diff
@@ -130,6 +130,70 @@ export class DataService {
     });
   }
 
+  /**
+   * Update curated feed item and the associated curated_feed_* tables
+   * @param eventBody
+   * @param curatedRecId
+   * @param resolvedId
+   * @param domainId
+   */
+  public async updateScheduledItem(
+    eventBody: ScheduledItemPayload,
+    curatedRecId: number,
+    resolvedId: number,
+    domainId: string
+  ) {
+    const topicId = await this.getTopicIdByName(
+      getTopicForReaditLaTmpDatabase(eventBody.topic)
+    );
+
+    const trx = await this.db.transaction();
```

Comment:
If you use the transaction handler, you don't have to call commit and rollback (it does this automatically). I have some updated handler function code to do error reporting, so we don't need to put all the event data in this error anymore.

Reply:
did this in order to throw a specific error, I can update if that is not a requirement.